app/build.gradle.kts

```diff
@@ -0,0 +1,226 @@
+        buildConfigField("String", "CREATE_ACCOUNT_URL", """"https://welcome.infomaniak.com/signup/ikdrive?app=true"""")
+        buildConfigField("String", "CREATE_ACCOUNT_SUCCESS_HOST", """"kdrive.infomaniak.com"""")
+        buildConfigField("String", "CREATE_ACCOUNT_CANCEL_HOST", """"welcome.infomaniak.com"""")
+        buildConfigField("String", "DRIVE_API_V1", """"https://api.kdrive.infomaniak.com/drive/"""")
+        buildConfigField("String", "DRIVE_API_V2", """"https://api.kdrive.infomaniak.com/2/drive/"""")
+        buildConfigField("String", "DRIVE_API_V3", """"https://api.kdrive.infomaniak.com/3/drive/"""")
+        buildConfigField("String", "MANAGER_URL", """"https://manager.infomaniak.com/v3/"""")
+        buildConfigField("String", "OFFICE_URL", """"https://kdrive.infomaniak.com/app/office/"""")
+        buildConfigField("String", "SHARE_URL_V1", """"https://kdrive.infomaniak.com/app/"""")
+        buildConfigField("String", "SHARE_URL_V2", """"https://kdrive.infomaniak.com/2/app/"""")
+        buildConfigField("String", "SHARE_URL_V3", """"https://kdrive.infomaniak.com/3/app/"""")
+        buildConfigField("String", "SHOP_URL", """"https://shop.infomaniak.com/order/"""")
+        buildConfigField("String", "SUPPORT_URL", """"https://support.infomaniak.com/"""")
+
+        resValue("string", "CLOUD_STORAGE_AUTHORITY", "com.infomaniak.drive.documents")
+        resValue("string", "FILE_AUTHORITY", "com.infomaniak.drive.files")
+
+        resValue("string", "EXPOSED_UPLOAD_DIR", "upload_media")
+        resValue("string", "EXPOSED_OFFLINE_DIR", "offline_storage")
+        resValue("string", "EXPOSED_PUBLIC_SHARE_DIR", "public_share")
+
+        androidResources.localeFilters += arrayOf("en", "de", "es", "fr", "it")
+    }
+
+    val javaVersion = JavaVersion.VERSION_17
+
+    compileOptions {
+        sourceCompatibility = javaVersion
+        targetCompatibility = javaVersion
+    }
+
+    kotlinOptions { jvmTarget = javaVersion.toString() }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+        }
+    }
+
+    flavorDimensions += "distribution"
+    productFlavors {
+        create("standard") {
+            apply(plugin = "com.google.gms.google-services")
+
+            isDefault = true
+
+            dependencies {
+                implementation("com.geniusscansdk:gssdk:5.6.1")
+            }
+        }
+        create("fdroid")
+        create("preprod") {
+            buildConfigField("String", "DRIVE_API_V1", """"https://kdrive.preprod.dev.infomaniak.ch/drive/"""")
+            buildConfigField("String", "DRIVE_API_V2", """"https://kdrive.preprod.dev.infomaniak.ch/2/drive/"""")
+            buildConfigField("String", "DRIVE_API_V3", """"https://kdrive.preprod.dev.infomaniak.ch/3/drive/"""")
+            matchingFallbacks += "standard"
+        }
+    }
+
+    gradle.buildFinished {
```

Comment:
This is deprecated.

Reply:
It already was, and there is no sane replacement, alas.
Good thing is it still works, and if/when it doesn't anymore, we can remove it with little impact if there's still no good replacement.